src/layouts/components/Editor/extensions/Cards/Cards.ts

```diff
@@ -154,9 +154,13 @@ export const IsomerCards = Node.create<CardOptions>({
         if (dispatch) {
           const offset = tr.selection.anchor + 1
 
-          tr.replaceSelectionWith(node)
+          tr.insert(tr.selection.anchor, node)
             .scrollIntoView()
-            .setSelection(Selection.near(tr.doc.resolve(offset)))
+            .setSelection(
+              Selection.near(
+                tr.doc.resolve(tr.doc.resolve(offset).parentOffset)
+              )
+            )
```

Comment:
could i get some context on why we need the parent offset? my understanding is that this is because our cards are nested components right?

Reply:
For some weird reason the selection will use one of the child elements (the image for one of the cards iirc) and not the card grid block, so opted to do this instead to ensure that we are selecting the card grid.